packages/fx-core/src/plugins/solution/fx-solution/debug/provisionLocal.ts

```diff
@@ -403,14 +403,18 @@ export async function configLocalDebugSettings(
         const botPassword = localSettings?.bot?.botPassword as string;
         botEnvs!.teamsfxLocalEnvs[EnvKeysBot.BotId] = botId;
         botEnvs!.teamsfxLocalEnvs[EnvKeysBot.BotPassword] = botPassword;
-        botEnvs!.teamsfxLocalEnvs[EnvKeysBot.ClientId] = clientId;
-        botEnvs!.teamsfxLocalEnvs[EnvKeysBot.ClientSecret] = clientSecret;
-        botEnvs!.teamsfxLocalEnvs[EnvKeysBot.TenantID] = teamsAppTenantId;
-        botEnvs!.teamsfxLocalEnvs[EnvKeysBot.OauthAuthority] = "https://login.microsoftonline.com";
-        botEnvs!.teamsfxLocalEnvs[EnvKeysBot.LoginEndpoint] = `${
-          localSettings?.bot?.botEndpoint as string
-        }/auth-start.html`;
-        botEnvs!.teamsfxLocalEnvs[EnvKeysBot.ApplicationIdUri] = applicationIdUri;
+
+        if (includeAAD) {
```

Comment:
Apply this change to `configLocalEnvironment` as well.
We are now merging localSettings to envInfo so temporarily code is duplicated. cc @xiaolang124